gemrb/override/shared/REVERB.2DA

```diff
@@ -0,0 +1,22 @@
+2DA V1.0
+-1
+		ENVIRONMENT	VOLUME	DECAY	DAMPING	REVERB_LEVEL
+NO_REVERB	GENERIC		0.0	0.0	0.0	0.0
+SMALL_ROOM	CITY		0.10	1.0	0.40	1.0
+MEDIUM_ROOM	AUDITORIUM	0.15	1.5	0.50	1.0
+LARGE_ROOM	AUDITORIUM	0.15	2.1	0.50	1.0
+OUTSIDE	        CITY		0.0	0.0	0.00	0.0
+DUNGEON	        AUDITORIUM	0.15	2.5	0.50	1.0
```

Comment:
these two are offset due mix of whitespace

Reply:
Raw extracted that ... I will fix this.